apps/core/templates/core/home-authenticated.html

```diff
@@ -22,6 +22,30 @@ <h2>{% trans "Welcome to Pixel" %}</h2>
       <div id="pixels-by-omics-unit-type" style="height: 500px"></div>
     </div>
 
+    <div class="cell">
+      <p style="font-family:'Arial'; font-size: 15px;font-weight: bold; color:black;text-align:center;">
+        {% trans "Who are Pixelers?" %}
+      </p>
+        <table style="margin: 0 auto;">
+          <thead>
+            <tr>
+              <th>Names</th>
+              <th>Email</th>
+              <th>Date joined</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for first_name,last_name, email, date in pixelers_infos.data %}
+            <tr>
+              <td>{{first_name}} {{last_name}}</td>
+              <td><a href="mailto:{{email}}" target="_top">{{email}}</a></td>
```

Comment:
Using `target` is not recommended on `<a>`, you should remove it.

Reply:
Also, format your code a bit when there is more than a word or two (or one or two variables)

```
<td>
  <a> ... </a>
</td>
```